docs/docs/topics/mutual-auth.md

```diff
@@ -0,0 +1,390 @@
+
+## Pomerium Defaults
+
+> - **Security**: Moderate
+> - **Difficulty**: Easy
+> - **Encrypted**: Yes (from the end user to Pomerium. It's up to the service to provide a TLS endpoint for Pomerium to use.)
+> - **Mutual Authentication**: None
+
+Let's look at a basic installation of Pomerium on a local network, with a single upstream service. This service contains sensitive data that we want to ensure is confidential and cannot be tampered with.
+
+::: tip Note
+Throughout this guide we will diagram Pomerium as a single service, as it is in all-in-one mode. This is the version provided by our [binaries] and in our Docker-based [Quick Start] page.
+:::
+
+```mermaid
+flowchart LR
+  C(End User)
+  D(Identity Provider)
+  C==o A
+  A-.-D
+  subgraph lan[Internal Network]
+    style lan stroke-dasharray: 5 5
+    A(Pomerium)
+    B(Upstream Service)
+    A --- B
+  end
+```
+
+1. When the user connects to the route `service.example.com` the traffic is received by Pomerium.
+1. Pomerium redirects the user to the identity provider (**IdP**) to sign in and validate their identity.
+1. After signing in to the IdP, Pomerium verifies that the authenticated (**authn**) user is authorized (**authz**) to access the service, and begins communication with it.
+
+This is a good start, only the users that are supposed to have access to the service get through. But this model is dependent on the security of your network perimeter. If a bad actor gains access to your internal network, they can now communicate with the service directly:
+
+```mermaid
+flowchart LR
+  C(End User)
+  C==o A
+  subgraph lan[Internal Network]
+    style lan stroke-dasharray: 5 5
+    A(Pomerium)
+    B(Upstream Service)
+    A --- B
+  end
+  E[/Hacker/]---B
+```
+
+While your network *should* be secured to only allow traffic at specified ports and directed to specified services, this creates a single point of failure. A hacker need only bypass your firewall to get direct access to your service.
+
+## JWT Verification: Application-based Mutual Authentication
+
+> - **Security**: Strong
+> - **Difficulty**: Moderate
+> - **Encrypted**: Yes (from the end user to Pomerium. It's up to the service to provide a TLS endpoint for Pomerium to use.)
+> - **Mutual Authentication**: Application Layer
+
+Many, but not all, modern web applications support [json web tokens][jwt-rfc] (**JWTs**). These tokens can be provided by Pomerium (with the [`pass_identity_headers`] key) so that an upstream service can independently verify that the traffic has properly gone through Pomerium, and that the incoming user request is authorized.
```

Comment:
```suggestion
Many, but not all, modern web applications support [JSON web tokens][jwt-rfc] (**JWTs**). These tokens can be provided by Pomerium (with the [`pass_identity_headers`] key) so that an upstream service can independently verify that the traffic has properly gone through Pomerium, and that the incoming user request is authorized.
```

I believe JSON should be capitalized. 

https://en.wikipedia.org/wiki/JSON

Reply:
There may be other instances I missed.